iotdb/src/main/java/org/apache/iotdb/db/engine/bufferwrite/BufferWriteProcessor.java

```diff
@@ -454,38 +485,6 @@ public long getFileSize() {
     return file.length() + memoryUsage();
   }
 
-  /**
-   * Close current TsFile and open a new one for future writes. Block new writes and wait until
-   * current writes finish.
-   */
-  public void rollToNewFile() {
-    // TODO : [MemControl] implement this
-  }
-
-  /**
-   * Check if this TsFile has too big metadata or file. If true, close current file and open a new
-   * one.
-   */
-  private boolean checkSize() {
```

Comment:
why you delete this function... 
I think this strategy is useful, and we need to consider where to call this method...

Reply:
This is done in another place.